packages/nuxt/src/app/plugins/check-outdated-build.client.ts

```diff
@@ -1,3 +1,4 @@
+import { FetchError } from 'ofetch'
```

Comment:
```suggestion
import type { FetchError } from 'ofetch'
```

Reply:
I think the instanceof test wouldn't work

Reply:
Ahh, you're probably right. My mistake!